src/unpage/cli/agent/serve.py

```diff
@@ -1,60 +1,64 @@
-import os
-
-import anyio
-import typer
-
 from unpage.agent.app import listen, settings
 from unpage.cli.agent._app import agent_app
 from unpage.telemetry import client as telemetry
 from unpage.telemetry import prepare_profile_for_telemetry
 
 
-@agent_app.command()
-def serve(
-    host: str = typer.Option(settings.UNPAGE_HOST, help="The host to bind to"),
-    port: int = typer.Option(settings.UNPAGE_PORT, help="The port to bind to"),
-    workers: int = typer.Option(settings.UNPAGE_WORKERS, help="The number of workers to use"),
-    profile: str = typer.Option(settings.UNPAGE_PROFILE, help="The profile to use"),
-    reload: bool = typer.Option(
-        settings.UNPAGE_RELOAD, help="Reload the server when the code changes"
-    ),
-    tunnel: bool = typer.Option(settings.UNPAGE_TUNNEL, help="Tunnel the server through ngrok"),
-    ngrok_token: str = typer.Option(
-        os.getenv("NGROK_TOKEN"),
-        help="The ngrok token to use to tunnel the server",
-        show_default=False,
-    ),
-    ngrok_domain: str = typer.Option(
-        settings.NGROK_DOMAIN, help="The ngrok domain to use to tunnel the server"
-    ),
+@agent_app.command
+async def serve(
+    *,
+    host: str = settings.UNPAGE_HOST,
+    port: int = settings.UNPAGE_PORT,
+    workers: int = settings.UNPAGE_WORKERS,
+    profile: str = settings.UNPAGE_PROFILE,
+    reload: bool = settings.UNPAGE_RELOAD,
+    tunnel: bool = settings.UNPAGE_TUNNEL,
+    ngrok_token: str = settings.NGROK_TOKEN,
```

Comment:
how about filtering what is shown as default in the help output?

```suggestion
    ngrok_token: Annotated[str, Parameter(show_default=lambda v: v if not v else f"{v[:3]}<redacted>")] = settings.NGROK_TOKEN,
```

Reply:
Oh that's cool, thanks!